src/run.jl

```diff
@@ -195,35 +195,68 @@ function init_simulation()
     s *= "| | AMR interval:     $amr_interval\n"
     s *= "| | adapt ICs:        $(adapt_initial_conditions ? "yes" : "no")\n"
   end
-  s *= """| n_steps_max:        $n_steps_max
-          | time integration:   $(get_name(time_integration_function))
-          | restart interval:   $restart_interval
-          | solution interval:  $solution_interval
-          | #parallel threads:  $(Threads.nthreads())
-          |
-          | Solver
-          | | solver:           $solver_name
-          | | polydeg:          $polydeg
-          | | CFL:              $cfl
-          | | volume integral:  $(get_name(solver.volume_integral_type))
-          | | volume flux:      $(get_name(solver.volume_flux_function))
-          | | surface flux:     $(get_name(solver.surface_flux_function))
-          | | #elements:        $(solver.n_elements)
-          | | #interfaces:      $(solver.n_interfaces)
-          | | #boundaries:      $(solver.n_boundaries)
-          | | #l2mortars:       $(solver.n_l2mortars)
-          | | #DOFs:            $(ndofs(solver))
-          |
-          | Mesh
-          | | #cells:           $(length(mesh.tree))
-          | | #leaf cells:      $n_leaf_cells
-          | | minimum level:    $min_level
-          | | maximum level:    $max_level
-          | | domain center:    $(join(domain_center, ", "))
-          | | domain length:    $domain_length
-          | | minimum dx:       $min_dx
-          | | maximum dx:       $max_dx
-          """
+  if ndims_ > 1
```

Comment:
Why do you need this extra check on ndims_ > 1?

Reply:
Because solver.n_l2mortars is not defined for 1D. Is there another way to solve this?